components/collator/src/lib.rs

```diff
@@ -214,10 +214,24 @@
 //!                     &Default::default(),
 //!                     options).unwrap();
 //!
-//! assert_eq!(primary_and_case.compare("DEAL", "ⓓⓔⓐⓛ"), Ordering::Equal);
-//! assert_eq!(primary_and_case.compare("ⓓⓔⓐⓛ", "déjavu"), Ordering::Less);
-//! assert_eq!(primary_and_case.compare("déjavu", "dent"), Ordering::Less);
-//! assert_eq!(primary_and_case.compare("dent", "develop"), Ordering::Less);
+//! assert_eq!(primary_and_case.compare("ⓓⓔⓐⓛ", "DEAL"), Ordering::Equal);
+//! assert_eq!(primary_and_case.compare("DEAL", "dejavu"), Ordering::Less);
+//! assert_eq!(primary_and_case.compare("dejavu", "déjavu"), Ordering::Equal);
+//! assert_eq!(primary_and_case.compare("déjavu", "dejAvu"), Ordering::Equal);
+//!
+//! // Secondary with case level on
+//!
+//! options.set_strength(Some(Strength::Secondary));
+//! options.set_case_level(Some(true));
+//! let secondary_and_case =
+//!   Collator::try_new_with_buffer_provider(&data_provider,
+//!                     &Default::default(),
+//!                     options).unwrap();
+//!
+//! assert_eq!(secondary_and_case.compare("ⓓⓔⓐⓛ", "DEAL"), Ordering::Equal);
+//! assert_eq!(secondary_and_case.compare("DEAL", "dejavu"), Ordering::Less);
+//! assert_eq!(secondary_and_case.compare("dejavu", "dejAvu"), Ordering::Equal);
+//! assert_eq!(secondary_and_case.compare("dejAvu", "déjavu"), Ordering::Less);  // secondary difference
```

Comment:
I would prefer, for better comparison among the options, to keep the example string pairs the same between each settings group, at least within a section like "Case Level".